src/black/mode.py

```diff
@@ -148,6 +148,7 @@ class Preview(Enum):
     one_element_subscript = auto()
     annotation_parens = auto()
     long_docstring_quotes_on_newline = auto()
+    remove_def_trailing_newline = auto()
```

Comment:
```suggestion
    remove_block_trailing_newline = auto()
```
NIT: If we want to be fully, pedantically accurate 😄